### PR TITLE
Lb testcases

### PR DESCRIPTION
**Pull request by KunTares**
Added testcases lb.py, lb_gpu.py and lb_stokes_sphere.py.
Partial fix for lbboundaries force interface.

### CODE REVIEW

**Comment by rempferg** [comment]
The test case lb.py requires a file lb_system.data, which seems to be missing.

File: testsuite/python/lb.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+		system.periodicity = [1, 1, 1]
+        	system.time_step = time_step
+        	system.cell_system.skin = skin
+
+		#  Clear actors that might be left from prev tests
+		if len(system.actors):
+			del system.actors[0]
+		system.part.clear()
+		#import particle data
+		data = np.genfromtxt("lb_system.data")
```
Comment (rempferg) [inline testsuite/python/lb.py]: Please add this file.

**Review by rempferg — CHANGES_REQUESTED** [review]
- I added all existing test cases to make check and corrected that one erroneous comma
- I removed erroneous integrate.integrate(100) lines from lb_test*.py
- I added all existing test cases to make check and corrected that one erroneous comma
- added ut decorators to only run minimize_energy test if LJ is compiled in

TODO
- add lb_system.data
- lb_test*.py do not contain asserts
- add decorators to only run LB tests of respective LB features compiled in

File: testsuite/python/lb.py [hunk 2]
```diff
@@ -12,6 +12,10 @@
 import numpy as np
 
 
+
+@ut.skipIf('LB' not in espressomd.code_info.features(),
```
Comment (hmenke) [inline testsuite/python/lb.py]: Please use `has_features`.  The `code_info` is not a user module.
````python
@ut.skipIf(not espressomd.has_features(["LB"]),
           "Features not available, skipping test!")
````

File: testsuite/python/lb_gpu.py [hunk 3]
```diff
@@ -11,7 +11,8 @@
 from espressomd import *
 import numpy as np
 
-
+@ut.skipIf('LB_GPU' not in espressomd.code_info.features(),
```
Comment (hmenke) [inline testsuite/python/lb_gpu.py]: Please use `has_features`.  The `code_info` is not a user module.
````python
@ut.skipIf(not espressomd.has_features(["LB_GPU"]),
           "Features not available, skipping test!")
````

File: testsuite/python/lb.py [hunk 4]
```diff
@@ -56,7 +60,7 @@ def test(self):
 			q = particle[2]
```
Comment (hmenke) [inline testsuite/python/lb.py]: `q` is now unused.  Please remove.

File: testsuite/python/lb_gpu.py [hunk 5]
```diff
@@ -56,7 +57,7 @@ def test(self):
 			q = particle[2]
```
Comment (hmenke) [inline testsuite/python/lb_gpu.py]: `q` is now unused. Please remove.

**Review by hmenke — CHANGES_REQUESTED** [review]

**Review by hmenke — APPROVED** [review]

**Review by rempferg — APPROVED** [review]

**Comment by rempferg** [comment]
I don't understand how the particle temperature can be 2 here. The calculation seems correct to me. Also, I can't reproduce this locally.

**Comment by fweik** [comment]
@rempferg @KunTares please check again. If fixed the two thermostat tests, and add the gpu stokes sphere test, which seemed ok.

**Comment by rempferg** [comment]
Looks good to me. Unfortunately I don't see a way to make the calculation of the fluid statistics faster without significant work.

**Review by rempferg — APPROVED** [review]

**Pull request merged**